<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogClass</class>
 <widget class="QDialog" name="DialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JAR Creator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Files</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Midlet name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="midletNameEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Input image:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputImageEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseInputImageBtn">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Input index (blank for none):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputIndexEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseInputIndexBtn">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editIndexBtn">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Output file:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="outputFileEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseOutputFileBtn">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Format:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="PNGRadio">
            <property name="text">
             <string>PNG</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string>formatButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="JPEGRadio">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>JPEG</string>
            </property>
            <attribute name="buttonGroup">
             <string>formatButtonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Tile size:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="tileSizeSpin">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="suffix">
             <string> pixels</string>
            </property>
            <property name="minimum">
             <number>20</number>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>80</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Zoom factor:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="zoomSpin">
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>8</number>
            </property>
            <property name="value">
             <number>4</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="iconBtn">
          <property name="text">
           <string>Icon...</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Storage method:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="jarRadio">
            <property name="text">
             <string>JAR</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string>storageButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="externalRadio">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>External</string>
            </property>
            <attribute name="buttonGroup">
             <string>storageButtonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="jadFileCheck">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Write JAD file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="usePNGCrushCheck">
          <property name="text">
           <string>Crush PNGs (requires 'pngcrush' in path)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>57</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>inputImageEdit</tabstop>
  <tabstop>browseInputImageBtn</tabstop>
  <tabstop>inputIndexEdit</tabstop>
  <tabstop>browseInputIndexBtn</tabstop>
  <tabstop>outputFileEdit</tabstop>
  <tabstop>browseOutputFileBtn</tabstop>
  <tabstop>PNGRadio</tabstop>
  <tabstop>JPEGRadio</tabstop>
  <tabstop>tileSizeSpin</tabstop>
  <tabstop>zoomSpin</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogClass</receiver>
   <slot>makeMidlet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseInputImageBtn</sender>
   <signal>clicked()</signal>
   <receiver>DialogClass</receiver>
   <slot>browseInputImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseInputIndexBtn</sender>
   <signal>clicked()</signal>
   <receiver>DialogClass</receiver>
   <slot>browseInputIndex()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseOutputFileBtn</sender>
   <signal>clicked()</signal>
   <receiver>DialogClass</receiver>
   <slot>browseOutputFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iconBtn</sender>
   <signal>clicked()</signal>
   <receiver>DialogClass</receiver>
   <slot>browseIcon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editIndexBtn</sender>
   <signal>clicked()</signal>
   <receiver>DialogClass</receiver>
   <slot>editIndex()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseInputImage()</slot>
  <slot>browseInputIndex()</slot>
  <slot>browseOutputFile()</slot>
  <slot>updateSizeEstimate()</slot>
  <slot>makeMidlet()</slot>
  <slot>browseIcon()</slot>
  <slot>editIndex()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="storageButtonGroup"/>
  <buttongroup name="formatButtonGroup"/>
 </buttongroups>
</ui>
